weasyprint/formatting_structure/build.py

```diff
@@ -235,6 +236,9 @@ def content_to_boxes(style, parent_box, quote_depth, counter_values,
                 counters.format(counter_value, counter_style)
                 for counter_value in counter_values.get(counter_name, [0])
             ))
+        elif type_ == 'string':
+            text = context.get_string_set_for(*tuple(value))
```

Comment:
Avoid crashing here when `context` is `None`. (Maybe just ignore `string()` functions.)